resources/configuration_file_template.yml

```diff
@@ -1,5 +1,6 @@
 s3_id: YOUR_AWS_S3_ACCESS_KEY_ID
 s3_secret: YOUR_AWS_S3_SECRET_ACCESS_KEY
+s3_token: YOUR_AWS_S3_SESSION_TOKEN # if using temporary credentials with a session token (eg: when assuming a role)
```

Comment:
This should probably be commented out by default, if we assume that most users will not use it. What do you think about this assumption?